tensorflow/compiler/tf2xla/kernels/bincount_op.cc

```diff
@@ -0,0 +1,162 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
We should not be adding new files on cherrypicks. Please find original source and patch that.